examples/01-multi-time-series-and-covariates.ipynb

```diff
@@ -167,13 +167,13 @@
    "metadata": {},
```

Comment:
<p>"""</p><ul><li>The&nbsp;<strong>target time series</strong>&nbsp;is the time series we are interested to forecast (given its history)</li></ul><p>"""</p><p>maybe some words about static covariates and a link to the static covariates notebook could help here as well</p><p> </p><p> </p><p>----</p><p> </p><p>"""</p><p>Some models use only past covariates, others use only future covariates, and some models might use both. We will dive deeper in this topic in some other notebook, but for now it is enough to know this:</p><p>"""</p><p> </p><p>the list below can also be replaced by adding a link to the readme with covariates support</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/2091/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>